client/Log.cpp

```diff
@@ -85,6 +85,15 @@ std::string FormatPacket(const OP2Internal::Packet& packet)
 	return ss.str();
 }
 
+std::string FormatLoadAddress(void* value)
+{
+	return FormatHexadecimal(reinterpret_cast<std::uintptr_t>(value));
+}
+
+std::string FormatLoadAddress(HINSTANCE value)
+{
+	return FormatHexadecimal(reinterpret_cast<std::uintptr_t>(value));
+}

```

Comment:
I believe this method is redundant. The `HINSTANCE` type is just a `typedef` for `void*`. If you remove this method, I believe the other remaining method would handle all existing calls. It does for Mingw-w64 at least.

Reply:
I just tested, and you are correct. I'll pull the HINSTANCE overload.